src/components/history-page/history-toolbar/history-toolbar.tsx

```diff
@@ -40,17 +47,20 @@ export interface HistoryToolbarProps {
   onSettingsChange: HistoryToolbarChange
 }
 
-export const initState: HistoryToolbarState = {
-  viewState: ViewStateEnum.CARD,
+const initSortState: ToolbarSortState = {
   titleSortDirection: SortModeEnum.no,
-  lastVisitedSortDirection: SortModeEnum.down,
+  lastVisitedSortDirection: SortModeEnum.down
+}
+
+export const initToolbarState: HistoryToolbarState = {
+  ...initSortState,
+  viewState: ViewStateEnum.CARD,
   keywordSearch: '',
-  selectedTags: []
+  selectedTags: [] as string[]
```

Comment:
why this additonal type cast?

Reply:
I had problems like this: https://stackoverflow.com/a/52423919

I'm unsure if these are still present and will recheck it.

Reply:
Fixed.